### ISSUE TITLE
create short links / tiny URLs for ease of sharing

### ISSUE THREAD

**Issue by cvan** [mozilla/hubs#336]
it's difficult to share URLs (or type on various devices) such as https://hubs.mozilla.com/0lnqkijxfny/glittering-yummy-adventure (especially when `l` and `I` look similar, and so on).

here's some sample JS code to use **https://mzl.la/**:

```js
process = {env: {BITLY_LOGIN: 'cvanmoz', BITLY_KEY: 'R_f3fbf18896b54f6c8c545b4ca7ecea68'}};

function bitly(url) {
  return fetch(`https://api-ssl.bitly.com/v3/shorten?login=${process.env.BITLY_LOGIN}&apiKey=${process.env.BITLY_KEY}&longUrl=${encodeURIComponent(url)}&format=txt`).then(res => res.text());
}

// NOTE: Mozilla has a bit.ly account (https://mzl.la), which is for only `*.mozilla.org` (not `*.mozilla.com`) URLs.
bitly('https://hubs.mozilla.org/0lnqkijxfny/glittering-yummy-adventure').then(res => { console.log(res); });
```

**Comment by gfodor** [comment]
Thanks for the suggestion + snippet but I don't think we want to use a shortener right now -- the ergonomics are certainly improved for some cases but there are some issues it introduces we decided overruled those conveniences:

- We become reliant upon `mzl.la` to facilitate co-present room joining, which is our core funnel and value prop. This significantly adds to our operability risk. I've suffered outages in a past life when relying upon bitly in particular :)

- The URLs we give users to share include the name of the room, which is important to us from a UX perspective.

- Having the app give users more than one URL go to the same virtual space is something we wanted to avoid semantically.

Perhaps there is a way to improve the ergonomics without introducing these problems?
